wordpress/wp-content/plugins/cds-base/classes/Modules/Blocks/src/styles/public.css

```diff
@@ -3,8 +3,9 @@ main {
 }
 
 /* Headings */
-.homepage h2 {
-  font-size: 1.2em;
+.homepage h2 ,
+.blog h2 {
```

Comment:
Just for reference what's this doing in this PR?

Reply:
The "index" page for me has the big headers again, which breaks the WET styling.